sqlassign2.sql

```diff
@@ -0,0 +1,91 @@
+Create a table students and insert names in Malayalam
+
+	create table students(id int identity(1,1) primary key,name nvarchar(50));
+	insert into students(name) values (N'ആലീസ്'),(N'അനു'),(N'വിജയ്');
+
+Retrieve all employees who work in Sales, Marketing, or IT departments.
+
+	select * from Employees where Department in ('Sales', 'Marketing','IT');
+
+Find all employees with salaries ranging from $50,000 to $75,000 (inclusive).
+
+	select * from Employees where Salary between 50000 and 75000;
+
+List all employees whose last name begins with the letter 'S'.
+
+	select * from Employees where LastName like 'S%';
+
+Display all employees with exactly five letters in their first name.
+
+	select * from Employees where FirstName like '_____';
+
+Find employees whose last name starts with either 'B', 'R', or 'S'.
+
+	select * from Employees where LastName like 'B%' or LastName like 'R%' or LastName like 'S%';
+
+Retrieve all employees whose first name begins with any letter from 'A' through 'M'.
+
+	select * from Employees where FirstName like '[A-M]%'
+
+List employees whose last name doesn't start with a vowel (A, E, I, O, U).
+
+	select * from Employees where LastName like '[^A,E,I,O,U]%'
+
+Identify employees earning more than $80,000 annually. 
+
+	select * from Employees where Salary>80000
+
+Find employees who joined the company before 2020.
+
+	select * from Employees where HireDate<'2020'
```

Comment:
Should we use year (HireDate) < '2020' here ?

Reply:
yes,sorry for the mistake.
Actually my column name had a typo error,so when i edit that in this file,the mistake happened.